### PR TITLE
#161342791 Setup an empty react project and host it on heroku

### PR DESCRIPTION
**Pull request by ucheg6**
#### What does this PR do?
Setup an empty react project
#### Description of Task to be completed?
- setup and configure webpack
- configure babel to transpile to es5
- configure eslint with airbnb styleguide

#### How should this be manually tested?
- Git clone the repository
- cd into the `haven-ah-frontend` folder
- run `npm run start:dev`
- To see the production build visit [`Authors Haven App`](https://haven-ah-frontend.herokuapp.com/)

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#161342791](https://www.pivotaltracker.com/story/show/161342791)
#### Screenshots (if appropriate)
screen shot 2018-11-26 at 3 25 18 pm
screen shot 2018-11-26 at 4 50 11 pm

#### Questions:
N/A

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      {
+        test: /\.css$/,
+        use: ['style-loader', 'css-loader'],
+      },
+      {
+        test: /\.(jpe?g|gif|png)$/,
+        loader: 'file-loader?name=images/[name].[ext]',
+      },
+      {
+        test: /\.js$/,
```
Comment (seyi-adeleke) [inline webpack.config.js]: please remove this

File: package.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    "babel-preset-env": "^1.7.0",
+    "babel-preset-react": "^6.24.1",
+    "css-loader": "^1.0.1",
+    "eslint": "^5.9.0",
+    "eslint-config-airbnb": "^17.1.0",
+    "eslint-loader": "^2.1.1",
+    "eslint-plugin-import": "^2.14.0",
+    "eslint-plugin-jsx-a11y": "^6.1.2",
+    "eslint-plugin-react": "^7.11.1",
+    "file-loader": "^2.0.0",
```
Comment (seyi-adeleke) [inline package.json:40]: file-loader is unused

**Review by seyi-adeleke — CHANGES_REQUESTED** [review]
...

File: package.json (hunk 2 above)
Comment (ucheg6) [reply package.json:40]: This is being used to load images

Comment (seyi-adeleke) [reply package.json:40]: At this very moment is this being
 used?

File: webpack.config.prod.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+const merge = require('webpack-merge');
+const UglifyJsPlugin = require('uglifyjs-webpack-plugin');
+const common = require('./webpack.config.common.js');
+
+module.exports = merge(common, {
+  mode: 'production',
+  devtool: 'source-map',
```
Comment (seyi-adeleke) [inline webpack.config.prod.js]: we dont need source maps on prod

**Pull request merged**